src/server/api/schema.js

```diff
@@ -139,6 +139,11 @@ const rootSchema = `
     userId: String
   }
 
+  input InviteInput {
+    id: String
```

Comment:
Empty input object throws and error. Just {is_valid: true} works, probably bc that's the only required attribute on the invite model https://github.com/MoveOnOrg/Spoke/pull/40/commits/36843534f9600f16d7d1f96d75c7106cc328b7a7